pkg/install/v1alpha1/cstor_sparse_pool_claim_0.7.0.go

```diff
@@ -48,10 +48,10 @@ apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
   name: openebs-cstor-sparse
+  labels:
+    openebs.io/cas-type: cstor
   annotations:
-    cas.openebs.io/create-volume-template: cstor-volume-create-default-0.7.0
-    cas.openebs.io/delete-volume-template: cstor-volume-delete-default-0.7.0
-    cas.openebs.io/read-volume-template: cstor-volume-read-default-0.7.0
+    openebs.io/cas-type: cstor
```

Comment:
Why do we need to have both label and annotation ?

Reply:
make sense. removed from labels. annotation is the one used by the code to map type to templates.